samcli/commands/remote/remote_invoke_context.py

```diff
@@ -92,8 +92,8 @@ def run(self, remote_invoke_input: RemoteInvokeExecutionInfo) -> None:
             DefaultRemoteInvokeResponseConsumer(self.stdout),
             DefaultRemoteInvokeLogConsumer(self.stderr),
         )
-        if not remote_invoke_executor:
-            raise NoExecutorFoundForRemoteInvoke(
+        if not remote_invoke_executor or self._resource_summary.resource_type not in SUPPORTED_SERVICES.values():
```

Comment:
nit: do we need second part of the `if` statement here? If there is no executor then we should be already failing here.

Reply:
:thinking_face: I was thinking with this check, we can add StepFunctions to the `REMOTE_INVOKE_EXECUTOR_MAPPING` dict in the factory and only maintain the supported_services dict until SFN support is released. But I agree this check here is redundant, I can remove SFN from  `REMOTE_INVOKE_EXECUTOR_MAPPING` and the 2nd part of the check.